### PR TITLE
[docs] Minor fixes to HACKING.md

### PR DESCRIPTION
**Pull request by aravindhp**
- Fix spacing
- Fix capitalization and short form usage
- Fix incorrect link to Git submodule section

### CODE REVIEW

File: docs/HACKING.md [hunk 1]
```diff
@@ -4,10 +4,10 @@
...
 - [Cluster and OS pre-requisites](wmco-prerequisites.md)
 - [Install](https://sdk.operatorframework.io/docs/installation/) operator-sdk v1.6.1
-- The operator is written using operator-sdk 
-[v1.6.1](https://github.com/operator-framework/operator-sdk/releases/tag/v1.6.1) and has the same 
-[pre-requisites](https://sdk.operatorframework.io/docs/installation/#prerequisites) as it does.
-- git submodules need to be initialized and up to date Before running build command. Pls refer [Updating Git submodules](Updating Git submodules)
+- The operator is written using operator-sdk [v1.6.1](https://github.com/operator-framework/operator-sdk/releases/tag/v1.6.1)
+  and has the same [pre-requisites](https://sdk.operatorframework.io/docs/installation/#prerequisites) as it does.
+- git submodules need to be initialized and up to date before running the build command. Please refer to
+  [Updating Git submodules](Updating Git submodules).
```
Comment (jrvaldes) [inline docs/HACKING.md:10]: Consider fixing the unresolved reference.

```suggestion
  [Updating Git submodules](#updating-git-submodules).
```

**Review by jrvaldes — COMMENTED** [review]
@aravindhp  Thanks for working on this, comment added. PTAL

File: docs/HACKING.md (hunk 1 above)
Comment (aravindhp) [reply docs/HACKING.md]: Fixed. PTAL.

**Pull request merged**